app/airq/commands/quality.py

```diff
@@ -50,12 +54,14 @@ class GetQuality(BaseQualityCommand):
     def _get_message(self, zipcode: Zipcode) -> typing.List[str]:
         message = []
         aqi = zipcode.aqi
+        aqi_display = gettext(" (AQI $(aqi)s)", aqi) if aqi else ""
         message.append(
-            "{} {} is {}{}.".format(
-                zipcode.city.name,
-                zipcode.zipcode,
-                zipcode.pm25_level.display,
-                f" (AQI {aqi})" if aqi else "",
+            gettext(
```

Comment:
We should make sure a string like `San Francisco 94118 es BUENO` makes sense in Spanish.

Reply:
Can't we translate it to anything? It doesn't have to be word for word. 
`San Francisco tiene buena calidad de aire`
`San Francisco esta BIEN`
I'll have to see when we extract though.

Reply:
Yeah we could, but since `pm25_level.display` is itself translated separately, it will be hard for the translator to have that context. We should make sure that whoever is translating that code understands that they're translating AQI levels instead of just strings like `good`.